integration-tests/benchmark/tests/keeper_test.go

```diff
@@ -187,6 +192,11 @@ var _ = Describe("Keeper benchmark suite @benchmark-keeper", func() {
 					Simulated:   benchmarkNetwork.Simulated,
 					WsURLs:      benchmarkNetwork.URLs,
 				}))
+			for _, version := range testEntry.registryVersions {
+				if version == eth_contracts.RegistryVersion_2_0 {
+					NumberOfNodes++
```

Comment:
umm, is this for bootstrap node?

Reply:
Yes. The number of nodes can be set via a test input. Since the test works for both non ocr and ocr keepers, this is done to keep the count test input consistent.